src/PerfView/CommandProcessor.cs

```diff
@@ -1344,6 +1346,7 @@ public void Stop(CommandLineArgs parsedArgs)
 
             DateTime stopComplete = DateTime.Now;
             LogFile.WriteLine("Stop Completed at {0}", stopComplete);
+            RundownActive = false;
```

Comment:
I would recommend doing this in a finally block, and put the rest of the method in a try block. This ensures that the value of `RundownActive` doesn't become stale if an exception is thrown, which can happen in this path.